### PR TITLE
414 allow models to be piped

### PR DESCRIPTION
**Pull request by mattfidler**

### CODE REVIEW

**Comment by mattfidler** [comment]
For me on ubuntu, the `devtools::check()` works just fine...

**Comment by mattfidler** [comment]
I can't get it to crash, perhaps it ran out of memory somewhere...

File: R/piping-ini.R [hunk 1]
```diff
@@ -347,20 +346,22 @@
...
   }
 
   lhs <- as.character(expr[[2]])
   wLhs <- which(ini$name == lhs)
   if (length(wLhs) != 1) {
-    stop("Cannot find parameter '", lhs, "'", call.=FALSE)
+    stop("cannot find parameter '", lhs, "'", call.=FALSE)
   } else if (length(appendClean) != 1) {
-    stop("Cannot find parameter '", after, "'", call.=FALSE)
+    stop("cannot find parameter '", append, "'", call.=FALSE)
```
Comment (mattfidler) [inline R/piping-ini.R:357]: @billdenney is my reading of this code correct, it should be append, right?

Comment (billdenney) [reply R/piping-ini.R:357]: That's right

**Pull request merged**